eclair-core/src/test/scala/fr/acinq/eclair/db/PaymentsDbSpec.scala

```diff
@@ -481,7 +481,7 @@ class PaymentsDbSpec extends AnyFunSuite {
       db.updateOutgoingPayment(PaymentFailed(s3.id, s3.paymentHash, Nil, 310))
       val ss3 = s3.copy(status = OutgoingPaymentStatus.Failed(Nil, 310))
       assert(db.getOutgoingPayment(s3.id) === Some(ss3))
-      db.updateOutgoingPayment(PaymentFailed(s4.id, s4.paymentHash, Seq(LocalFailure(Seq(hop_ab), new RuntimeException("woops")), RemoteFailure(Seq(hop_ab, hop_bc), Sphinx.DecryptedFailurePacket(carol, UnknownNextPeer))), 320))
+      db.updateOutgoingPayment(PaymentFailed(s4.id, s4.paymentHash, Seq(LocalFailure(500 msat, Seq(hop_ab), new RuntimeException("woops")), RemoteFailure(400 msat, Seq(hop_ab, hop_bc), Sphinx.DecryptedFailurePacket(carol, UnknownNextPeer))), 320))
```

Comment:
Where do these values come from? I can't link them to any of the other values above.

Reply:
I clarified it in https://github.com/ACINQ/eclair/pull/1966/commits/fd216f2abbd0a1163197e7915eca8cccb25c2465 (I used the amount from the right payment)